.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_KEY = AIzaSyAZYb0hiNlAZgqa86GtqvDsddkag2e5UfQ
```

Comment:
never ever ever commit .env file. Always create a .gitignore and add .env in there